sal_interfaces/MTMount/MTMount_Telemetry.xml

```diff
@@ -6,291 +6,129 @@
-          <Description>ACW_Current_Drive_1</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>ACW_Current_Drive_2</EFDB_Name>
-          <Description>ACW_Current_Drive_2</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>ACW_AngleDif_Actual</EFDB_Name>
-          <Description>ACW_AngleDif_Actual</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>ACW_AngleDif_ActualLVDT</EFDB_Name>
-          <Description>ACW_AngleDif_ActualLVDT</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>ACW_Angle_Setpoint</EFDB_Name>
-          <Description>ACW_Angle_Setpoint</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>deg</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>ACW_Positive_Power_Off_switch</EFDB_Name>
-          <Description>ACW_Positive_Power_Off_switch</Description>
-          <IDL_Type>boolean</IDL_Type>
-          <Units>unitless</Units>
+          <Units>   </Units>
           <Count>1</Count>
       </item>
      <item>
-          <EFDB_Name>ACW_Negative_Power_Off_switch</EFDB_Name>
-          <Description>ACW_Negative_Power_Off_switch</Description>
-          <IDL_Type>boolean</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>ACW_Interlocks</EFDB_Name>
-          <Description>ACW_Interlocks</Description>
-          <Size_in_bytes>1</Size_in_bytes>
+          <EFDB_Name>AXES_PXI_Version</EFDB_Name>
+          <Description>AXES_PXI_Version</Description>
+          <IDL_Size>1</IDL_Size>
           <IDL_Type>string</IDL_Type>
-          <Units>unitless</Units>
-          <Count>10</Count>
-      </item>
-     <item>
-          <EFDB_Name>timestamp</EFDB_Name>
-          <Description>timestamp</Description>
-          <Size_in_bytes>1</Size_in_bytes>
-          <IDL_Type>double</IDL_Type>
-          <Units>second</Units>
+          <Units>   </Units>
           <Count>1</Count>
       </item>
-</SALTelemetry>
-<SALTelemetry>
- <Subsystem>MTMount</Subsystem>
- <Version>3.2.0</Version>
- <Author>Ander Ansuategi</Author>
- <EFDB_Topic>MTMount_Azimuth</EFDB_Topic>
      <item>
-          <EFDB_Name>Azimuth_Status</EFDB_Name>
-          <Description>Azimuth_Status</Description>
-          <Size_in_bytes>1</Size_in_bytes>
+          <EFDB_Name>TMA_PXI_Version</EFDB_Name>
+          <Description>TMA_PXI_Version</Description>
+          <IDL_Size>1</IDL_Size>
           <IDL_Type>string</IDL_Type>
-          <Units>unitless</Units>
+          <Units>   </Units>
```

Comment:
Units cannot be blank or just whitespace.  Please use `unitless` or `dimensionless`.